test/util/images/images.go

```diff
@@ -0,0 +1,36 @@
+package images
+
+import (
+	"fmt"
+)
+
+var (
+	registry  string
+	testImage string
+)
+
+func init() {
+	registry = "docker.io"
+	testImage = "alpine"
+}
+
+// GetPodTestImage returns image to be used during testing
+func GetPodTestImage() string {
+	return fmt.Sprintf("%s/%s", registry, testImage)
+}
+
+// SetImageRegistry - set image registry
+func SetImageRegistry(imgRegistry string) {
+	registry = imgRegistry
+}
+
+// SetTestImageName - set test image name
+func SetTestImageName(name string) {
+	testImage = name
+}
+
+// ResetToDefaults - reset values to default state
+func ResetToDefaults() {
```

Comment:
`ResetToDefaults(...)` is not actually invoked. Anyway, I think there isn't a real need for a stateful package. Maybe it's better to add a string parameter to `GetPodDefinition(...)` with the test image or put a hard-coded one, since it will be used -probably- always for network testing. What do you think?

Reply:
Take for example NRI:
https://github.com/k8snetworkplumbingwg/network-resources-injector/blob/master/test/util/images.go

We use this images.go package.

I think it is useful to have a separated package for future testing.
The `ResetToDefaults` func, good catch that it is not being called, it may still be useful for future additional tests which may need to change the image, and then later revert back to default.
WDYT?